fe/lib/components/LocationSuggest/components/queries.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from '@apollo/client';
+
+export const NEAREST_LOCATIONS = gql`
+  query ($geoLocation: GeoLocationInput!, $schemeId: String!) {
```

Comment:
To my point above, I think you will have to name this query in order to get a usable codegenned type. It looks like we have a mix of upper and lower camel cased names across this codebase 😬